sdk/keyvault/azure_security_keyvault_keys/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) Rust Code Generator. DO NOT EDIT.
+
+mod generated;
+
+pub mod clients {
+    pub use crate::generated::clients::*;
+}
+
+pub mod models {
+    pub use crate::generated::enums::*;
+    pub use crate::generated::models::*;
+}
+
+pub use crate::generated::clients::{
```

Comment:
@jhendrixMSFT I thought we were also putting request-only and request-response models in here as well.

Reply:
From our conversation last Friday we decided to punt on that for now.